src/Git.hs

```diff
@@ -99,6 +106,9 @@ instance RefSpec Sha where
 instance RefSpec (Sha, Branch) where
   refSpec (sha, remote) = refSpec sha ++ ":refs/heads/" ++ refSpec remote
 
+instance RefSpec (Sha, BaseBranch) where
```

Comment:
Is the other one (`(Sha, Branch)`) still used then?

Reply:
It is used here: https://github.com/channable/hoff/blob/abbf862e9d9c46c1cce64f4c6fae3fe2e37b112b/src/Git.hs#L286-L296

I think `ForcePush` is used for the testing branch as well, so I can't change the `Branch` to `BaseBranch`.

Reply:
:ok_hand: